docs/reference/glossary.md

```diff
@@ -53,7 +56,7 @@ See [Accounts](../concepts/accounts.md) for more information.
 
 ### Aptos-core
 
-**Aptos-core** is the open source technology on which the Aptos Payment Network runs. Aptos-core contains software for
+**Aptos-core** is the [open source technology](https://github.com/aptos-labs/aptos-core/) on which the Aptos Network runs. Aptos-core contains software for
```

Comment:
Ugh... this is also bad.

`aptos-core` is the open-source repository containing the code for the Aptos Network software.